tests/unit/lms/services/course_test.py

```diff
@@ -109,24 +106,13 @@ def svc(self, pyramid_request):
 
     @pytest.fixture(autouse=True)
     def application_instance(self, pyramid_request):
-        return self._make_application_instance(
-            pyramid_request, pyramid_request.lti_user.oauth_consumer_key
+        return factories.ApplicationInstance(
+            consumer_key=pyramid_request.lti_user.oauth_consumer_key
         )
 
     @pytest.fixture
-    def other_application_instance(self, pyramid_request):
-        return self._make_application_instance(pyramid_request, consumer_key="other")
-
-    def _make_application_instance(self, pyramid_request, consumer_key):
-        application_instance = ApplicationInstance(
-            consumer_key=consumer_key,
-            shared_secret="test_shared_secret",
-            lms_url="test_lms_url",
-            requesters_email="test_requesters_email",
-        )
-        pyramid_request.db.add(application_instance)
-        pyramid_request.db.flush()
-        return application_instance
+    def other_application_instance(self):
+        return factories.ApplicationInstance()
```

Comment:
```suggestion
        return factories.ApplicationInstance(
            consumer_key="something_different"
        )
```

Is this why these aren't the same? This isn't technically required, as the factoryboy default is a randomized hex, but this makes it explicit why these aren't the same.

Reply:
:+1: